public/apps/configuration/panels/service-account-list.tsx

```diff
@@ -111,8 +114,14 @@ export function ServiceAccountList(props: AppDependencies) {
         );
         setCurrentUsername((await getAuthInfo(props.coreStart.http)).user_name);
         setUserData(await userDataPromise);
+        setErrorFlag(false);
+        setAccessErrorFlag(false);
       } catch (e) {
         console.log(e);
+        // requests with existing credentials but insufficient permissions result in 403, remote data-source requests with non-existing credentials result in 400
+        if (e.response && [400, 403].includes(e.response.status)) {
+          setAccessErrorFlag(true);
+        }
```

Comment:
Maybe shallow is not waiting for the component to re-render and show your custom component

Reply:
You may need to use mount instead: https://enzymejs.github.io/enzyme/docs/api/mount.html

Reply:
Personally I wouldn't block on this codecov, but good to verify the change

Reply:
> Maybe shallow is not waiting for the component to re-render and show your custom component

i don't think so, since tests would fail if component didn't render correctly. Its something else